### PR TITLE
Add the license information

### PR DESCRIPTION
**Pull request by optimumtact**
Fixes goofs mistake when porting wrestling

### CODE REVIEW

**Comment by ZeWaka** [comment]
Could we get a perhaps modified comment on https://github.com/tgstation/tgstation/blob/11069f4bbd0b195987b1d792922b54e3484fe977/code/datums/mutations/hulk.dm#L155 here as well, since it derived from this?

**Comment by optimumtact** [comment]
> Could we get a perhaps modified comment on
> 
> here as well, since it derived from this?

Do you want the same header, or a comment pointing to the wrestling file? Ideally we would refactor them into a shared proc to make it clean, but I dont' have time to do that too

**Review by Cyberboss — DISMISSED** [review]

File: code/datums/mutations/hulk.dm [hunk 1]
```diff
@@ -151,7 +151,13 @@
 	yeeted_person.emote("scream")
 	swing_loop(the_hulk, yeeted_person, 0, original_dir)
 
-/// For each step of the swinging, with the delay getting shorter along the way. Checks to see we still have them in our grasp at each step.
+/**
+ * Does the animations for the hulk swing loop
+ *
+ * This code is based in part on the wrestling swing code ported from goon, see [code/datums/martial/wrestling.dm]
+ * credi to: cogwerks, pistoleer, spyguy, angriestibm, marquesas, and stuntwaffle.
```
Comment (ZeWaka) [inline code/datums/mutations/hulk.dm:158]:
```suggestion
 * credit to: cogwerks, pistoleer, spyguy, angriestibm, marquesas, and stuntwaffle.
```
LGTM

**Review by ZeWaka — APPROVED** [review]

**Comment by LemonInTheDark** [comment]
Everything settled here?

**Comment by optimumtact** [comment]
yes

**Pull request merged**